prime-router/src/main/kotlin/azure/RequestFunction.kt

```diff
@@ -70,27 +73,40 @@ abstract class RequestFunction(
         routeTo.filter { workflowEngine.settings.findReceiver(it) == null }
             .forEach { actionLogs.error(InvalidParamMessage("Invalid receiver name: $it")) }
 
+        var sender: Sender? = null
```

Comment:
This abstract class, and specifically this method, is exercised via the tests that test its subclasses: ReportFunctionTests and ValidateFunctionTests. The new (and old) ValidateFunctionTests in particular exercise the code highlighted here, through their calls to `ValidateFunction.processRequest`. We currently do not have any direct unit tests for the abstract class RequestFunction, but if you would like me to test that method directly in this ticket I can.

Reply:
Also, I just simplified this code, moving almost all schema/format logic to `getDummySender` to reduce redundancy with the logic in ValidateFunction. So, I added tests for `getDummySender` since its more complicated now.